consensus_test.go

```diff
@@ -208,6 +130,96 @@ func TestTransition_AcceptState_Validator_VerifyFails(t *testing.T) {
 	})
 }
 
+func TestTransition_AcceptState_Proposer_FailedBuildProposal(t *testing.T) {
+	buildProposalFunc := func() (*Proposal, error) {
+		return nil, errors.New("failed to build a proposal")
+	}
+
+	validatorIds := []string{"A", "B", "C"}
+	backend := newMockBackend(validatorIds, nil, buildProposalFunc, nil, nil)
+
+	m := newMockPbft(t, validatorIds, "A", backend)
+	m.state.view = ViewMsg(1, 0)
+	m.setState(AcceptState)
+
+	var buf bytes.Buffer
```

Comment:
Remove this, this logger should be abstracted away in the mock pbft. I do not want to put 5 lines on each test.

Reply:
Ok, agreed. We just wanted to enforce assertions in order to be sure that expected part of code executed.